p2p_wallet/AppDelegate.swift

```diff
@@ -64,16 +65,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             window?.overrideUserInterfaceStyle = Defaults.appearance
         }
 
-        notificationService.wasAppLaunchedFromPush(launchOptions: launchOptions)
-
-        // set rootVC
-        let vm = Root.ViewModel()
-        let vc = Root.ViewController(viewModel: vm)
-        window?.rootViewController = vc
+        // add placeholder
+        window?.rootViewController = BaseVC()
```

Comment:
I think it's better to move window setting logic to coordinator and this code is going to look as this
`
appCoordinator.start()
window = appCoordinator.mainWindow
`